src/api/data/helm/releases/releases.go

```diff
@@ -0,0 +1,73 @@
+package releases
+
+import (
+	log "github.com/Sirupsen/logrus"
```

Comment:
We might want to standardize to `logrus` everywhere; we this introduction I think it's safe to say we have a heterogeneous logging implementation.